joomla/joomla-cms
# [4.2] Consider permissions for MM actions

Pull Request for Issue #36549.

### Summary of Changes
Passes the global permissions from the media manager to the client app. The item can define a flag as well to specify if the item itself can be edited/deleted. This allows external adapters to define also per item permissions.

This pr is based on the pr from @dgrammatiko and #36556.

### Testing Instructions
- Check the instructions in issue #36549.
- Additionally add a canEdit/canDelete flag to the file /plugins/filesystem/local/src/Adapter/LocalAdapter.php on line 344.

### Actual result BEFORE applying this Pull Request
Actions are always shown.

### Expected result AFTER applying this Pull Request
Actions are only shown when the permissions are correct.
